src/FNAPlatform/SDL3_FNAPlatform.cs

```diff
@@ -1031,6 +1031,24 @@ ref bool textInputSuppress
 					{
 						SDL.SDL_EnableScreenSaver();
 					}
+
+					// Full screen
+					else if (evt.type == (uint) SDL.SDL_EventType.SDL_EVENT_WINDOW_ENTER_FULLSCREEN)
+					{
+						Object manager = game.Services.GetService(typeof(IGraphicsDeviceManager));
+						if (manager is GraphicsDeviceManager)
+						{
+							((GraphicsDeviceManager) manager).IsFullScreen = true;
+						}
+					}
+					else if (evt.type == (uint) SDL.SDL_EventType.SDL_EVENT_WINDOW_LEAVE_FULLSCREEN)
+					{
+						Object manager = game.Services.GetService(typeof(IGraphicsDeviceManager));
+						if (manager is GraphicsDeviceManager)
```

Comment:
In case of weird timing issues:
```suggestion
						if (manager != null && manager is GraphicsDeviceManager)
						{
							((GraphicsDeviceManager) manager).IsFullScreen = true;
						}
					}
					else if (evt.type == (uint) SDL.SDL_EventType.SDL_EVENT_WINDOW_LEAVE_FULLSCREEN)
					{
						Object manager = game.Services.GetService(typeof(IGraphicsDeviceManager));
						if (manager != null && manager is GraphicsDeviceManager)
```

Reply:
Null check added